org.freedesktop.Piper.json

```diff
@@ -0,0 +1,60 @@
+{
+    "app-id": "org.freedesktop.Piper",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "piper",
+      "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--system-talk-name=org.freedesktop.ratbag1"
```

Comment:
Is the idea now to have ratbagd running in the system and allow the sandboxed Piper to communicate with it over DBus through `"--system-talk-name=org.freedesktop.ratbag1"`? This doesn't seem to be working for me at the moment.

Otherwise, with libratbag still requiring udev, aren't we in the same boat we were in in https://github.com/libratbag/piper/issues/5?